src/db.js

```diff
@@ -12,12 +12,14 @@ const db = [
 				date: '07-06-2022',
 				change: -40.31,
```

Comment:
Maybe it makes sense to have an `expense: negative number ` , `income: positive number`  🤔
That way your entries stay clean and more specific.

Reply:
Thanks for the advice. We where not sure how we want to implement this. We thought maybe through buttons or so.  
We will have a look 😃